hero/martial-artist-ken.json

```
@@ -0,0 +1,199 @@
+{
+    "name": "Martial Artist Ken",
+    "rarity": 5,
+    "classType": "warrior",
+    "element": "dark",
+    "zodiac": "leo",
+    "specialtyChangeName": "",
+    "selfSkillBarName": "",
+    "background": [
+        "In his goal of becoming the strongest in the univerise, he is hunting down and challenging every martial arts master he can find. He will cross time and space, and even dimensions, to find a worthy challenger."
```

```suggestion
        "In his goal of becoming the strongest in the universe, he is hunting down and challenging every martial arts master he can find. He will cross time and space, and even dimensions, to find a worthy challenger."
```
Small typo: `univerise -> universe`